util-core/src/main/scala/com/twitter/util/Bijection.scala

```diff
@@ -57,5 +57,5 @@ trait Bijection[A, B] extends (A => B) { self =>
   /**
    * Composes two instances of Bijection in a new Bijection, with this one applied last.
    */
-  def compose[T](g: Bijection[T, A]) = g andThen this
+  def compose[T](g: Bijection[T, A]): Bijection[T, B] = g andThen this
```

Comment:
we're getting rid of this really soon in exchange for twitter bijections, so let's make sure you merge master before we merge this in.